### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vfiftyfive**
- Updated readme to describe the current approach to creating network policies and include information on how DNS intents are resolved.
- Update the network policy and DNS sections of the README

### Description

- Update netwok policy section with correct YAML for netpol
- Reword content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,26 +47,103 @@ spec:
...
+```
+</details>
+
+For more usage example see the [network policy](https://docs.otterize.com/quick-tutorials/k8s-network-policies) and [egress policy automation](/features/network-mapping-network-policies/tutorials/k8s-egress-access-control-tutorial) tutorials.
+
+### Client intents and DNS
+
+When a client intents is specified using DNS identifiers, such as `api.exampledomain.com` in the example above, it initiates a sequence of operations between the [network mapper](https://github.com/otterize/network-mapper) and the intents operator to integrate the relevant IP addresses into the appropriate `NetworkPolicies`.
+
+1. The network mapper incorporates a DNS cache layer, which identifies and archives all resolved DNS records alongside their corresponding IPv4 and IPv6 IP addresses.
```
Comment (orishoshan) [inline README.md:142]:
```suggestion
1. The network mapper incorporates a DNS cache layer, which identifies and stores all resolved DNS records alongside their corresponding IPv4 and IPv6 IP addresses.
```

File: README.md [hunk 2]
```diff
@@ -47,26 +47,103 @@ spec:
...
-to match declarations in client intents files.
-The policies created are `Ingress`-based, so source pods are labeled with a `can-access-<target>=true` 
-while destination pods are labeled with `has-identity=<target>`.
+The intents operator automates the creation, management, and deletion of network policies based on user-defined client intents. To track the services it manages, it assigns labels to pods. These labels are derived from the pod's metadata, specifically combining the name of its top-level owner (e.g., a Kubernetes deployment) and the namespace where it's deployed. This unique identifier is then used within the generated network policies. The operator handles both **Ingress** and **Egress** traffic.
 
-The example above results in the following network policy being created: 
+In the network policy detailed below, we can see an example of Ingress definitions. Two separate policies are defined for the backend and frontend services within the `otterize-example` namespace, each designed to strictly control ingress traffic based on pod labels.
```
Comment (orishoshan) [inline README.md:52]: This is confusing to me as it's talking about Ingress definitions, but the immediate association is with an Ingress resource. Could we say a NetworkPolicy with type Ingress instead?

**Review by orishoshan — APPROVED** [review]

**Comment by orishoshan** [comment]
@vfiftyfive bump

**Comment by amitlicht** [comment]
@orishoshan @vfiftyfive closing this as it wasn't merged and has already decayed. Please re-open if relevant.

**Pull request closed without merging**